developer_manual/client_apis/WebDAV/bulkupload.rst

```diff
@@ -0,0 +1,43 @@
+===================
+File bulk upload
+===================
+
+.. sectionauthor:: Matthieu Gallien <matthieu.gallien@nextcloud.com>
+
+Introduction
+------------
+Uploading many small files is often slower than what could be achieved because we do not
+use the whole network bandwidth. Nextcloud has a bulk upload API where you can upload
+many small files together in order to optimize the use of network bandwidth.
+
+Usage
+-----
+
+The API is only available for registered users of your instance. And uses the path:
+``<server>/remote.php/dav/bulk``.
+
+Starting a bulk upload
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A bulk upload is simply using a request structured as HTTP multipart with related mime type.
```

Comment:
It it possible to have a cURL or general example for this?

Reply:
let me check if the example from @artonge are working against current server implementation

Reply:
added example code taken from server tests script